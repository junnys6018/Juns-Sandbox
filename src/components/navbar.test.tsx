import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Navbar from './navbar';

test('<Navbar /> snapshot', () => {
    const { asFragment } = render(<Navbar />, { wrapper: MemoryRouter });
    expect(asFragment()).toMatchSnapshot();
});
