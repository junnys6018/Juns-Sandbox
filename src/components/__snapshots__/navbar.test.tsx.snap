// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> snapshot 1`] = `
<DocumentFragment>
  <nav
    class="container flex flex-row py-10"
  >
    <a
      class="font-semibold text-3xl mr-auto bg-clip-text text-transparent bg-gradient-to-r from-[#ff0099] to-[#ff9900]"
      href="/"
    >
      Jun's Sandbox
    </a>
    <a
      class="font-medium text-xl text-pink-600"
      href="/"
    >
      Home
    </a>
  </nav>
</DocumentFragment>
`;
